### ISSUE TITLE
Update to Obsidian 1.0 broke styling of several elements

### ISSUE THREAD

**Issue by lodygaw** [colineckert/obsidian-things#94]
Hi!
I have noticed several elements whose styling has been broken by the recent Obsidian update. I made a list of elements I encountered so far:
- Kanban plugin (tags, lists, cards... almost everything looks different). See screenshots for reference:

Obsidian 0.16:
<img width="1420" alt="old" src="https://user-images.githubusercontent.com/55928022/195804874-7fdd570f-65ca-42ba-9a2a-d032cfecd118.png">
Obsidian 1.0:
<img width="1416" alt="Zrzut ekranu 2022-10-14 o 10 40 07" src="https://user-images.githubusercontent.com/55928022/195805008-91cfa86b-069a-4169-aea4-29353a58d043.png">

- note title (much bigger, aligned with text(shifted to the left previously))
- URLs (were gray, now accent color)
- the color of the text (I might be wrong but it seems "more black" in new version)
- "toolbox" in the upper right corner has a lighter grey shade

Obsidian 0.16:
<img width="1429" alt="Zrzut ekranu 2022-10-14 o 10 54 34" src="https://user-images.githubusercontent.com/55928022/195806553-c03828a4-69f9-4460-9976-47563f158e50.png">
Obsidian 1.0:
<img width="1412" alt="Zrzut ekranu 2022-10-14 o 10 55 31" src="https://user-images.githubusercontent.com/55928022/195806652-3b2b92de-f21f-4897-995e-b82242740bda.png">

Hope this will help with the fixes!

**Comment by universvm** [comment]
Agreed, as above, the title of the file is now displayed in the note as a gigantic H1. 

<img width="1223" alt="Screenshot 2022-10-14 at 14 54 16" src="https://user-images.githubusercontent.com/17524568/195851884-30b4dd77-f7e3-4d0a-bda2-ca50823e1177.png">

**Comment by colineckert** [comment]
Hey @lodygaw I recommend updating your theme. Yesterday I released Things v2 which is compatible with the UI changes introduced in Obsidian 1.0. Lmk your thoughts!

**Comment by yuukoyoung** [comment]
nice

**Comment by lodygaw** [comment]
@colineckert 
Thanks for the fast reply! 
I forgot to mention: I am already on v2.0.0, I updated immediately after I noticed that there is a new version available. So these changes I mentioned above are affecting the new version as well.

**Comment by colineckert** [comment]
hey, @lodygaw I just pushed up some updates to the kanban styling. Working on headers and fonts next. The reason for a lot of the changes you called out is that Things 2 was a complete rewrite since Obsidian v1 introduced a lot of new style schemes. So (slowly) bringing back some of the touches in Things v1.
